src/components/ControlBar.js

```diff
@@ -13,45 +13,45 @@ function ControlBar (): React.MixedElement {
   const buttonClicked = (e: SyntheticEvent<HTMLButtonElement>): void => {
     const modalType: string = e.currentTarget.name;
 
-    if (modalType === 'teamConfig') {
+    if (modalType === "teamConfig") {
       dispatch(toggleModal(!modalState));
     }
     // TODO: instantiate other types of modals based on button.name.
   };
 
   const controlBarStyle = {
-    display: 'flex',
-    width: '100%',
-    background: 'black',
-    height: '30%',
-    justifyContent: 'space-evenly',
+    display: "flex",
+    width: "100%",
+    background: "black",
+    height: "30%",
+    justifyContent: "space-evenly",
   };
 
   const TeamConfig: React.MixedElement = 
     <Button 
-      id={'team-config'}
-      className={'control-bar'}
-      name={'teamConfig'}
-      title={'Configurations'}
+      id="team-config"
+      className="control-bar"
+      name="teamConfig"
+      title="Configurations"
       action={buttonClicked}
     />;
 
   const Start: React.MixedElement = 
     <Button 
-      id={'start'}
-      className={'control-bar'}
-      name={'start'}
-      title={'Start'}
+      id="start"
+      className="control-bar"
+      name="start"
+      title="Start"
       action={buttonClicked}
     />;
 
 
   const Reset: React.MixedElement = 
     <Button 
-      id={'reset'}
-      className={'control-bar'}
-      name={'reset'}
-      title={'Reset'}
+      id="reset"
+      className="control-bar"
+      name="reset"
+      title="Reset"
       action={buttonClicked}
     />;
 
```

Comment:
last thing I forgot to mention is that we definitely prefer all the html markup inside the main return, helps readability. Unless you have a case where you are going to re-use the exact same markup more than once, then putting into a variable is fine